### PR TITLE
challenges: update submission requests for challenges that are in the…

### PR DESCRIPTION
**Pull request by bekcpear**
… extended championship

### CODE REVIEW

**Review by sunmin89 — COMMENTED** [review]
增加了一些说明，方便组委会验证

File: docs/challenges/P2421/index.mdx [hunk 1]
```diff
@@ -38,14 +38,11 @@ import ChallengeRegistration from '../../../src/components/ChallengeRegistration
...
-- **若未以源码形式提交，比赛结果公布后，需对提交内容进行完整开源才算完成本赛题。**
-- 提交方式可以有：
-  - 二进制的形式
-  - 加密源码的形式（加密信息需通过邮件发送至 rvspoc@kubuds.cn）
-  - 源码形式
+1. 提交仓库为 https://github.com/rv2036/rvspoc-P2421-seL4
+2. 请以 Pull request 的形式，将结果提交到上述仓库中。
+   - 以下条件中的「PR」均指代相对应于条件 1 仓库链接的 Pull request。
+3. 请在创建 PR 时，提供用于满足移植任务完成的必要内容（包括程序或相应补丁；以及说明信息）。
+4. **在比赛时间截止前，需通过 PR 的方式对提交内容进行完整开源才算完成本赛题。**
```
[the remark this answers is not in the document]
Comment (bekcpear) [reply docs/challenges/P2421/index.mdx:45]: 可以对 [update-challenges](https://github.com/rv2036/rvspoc/tree/update-challenges) 分支进行提交以进行此赛题的相应修改

[the remark this answers is not in the document]
Comment (sunmin89) [reply docs/challenges/P2421/index.mdx]: 好

**Review by shiptux — APPROVED** [review]
LGTM

**Pull request merged**